iocore/net/SSLClientUtils.cc

```diff
@@ -82,6 +82,11 @@ verify_callback(int signature_ok, X509_STORE_CTX *ctx)
       } else {
         sni_name = buff;
       }
+      if (is_debug_tag_set("ssl_verify")) {
+        Debug("ssl_verify", "Core server certificate verification failed for (%s). Action=%s Error=%s server=%s(%s) depth=%d",
+              sni_name, enforce_mode ? "Terminate" : "Continue", X509_verify_cert_error_string(err),
+              netvc->options.ssl_servername.get(), buff, depth);
+      }
```

Comment:
Are you wrapping these in the conditional because the computation of the parts of the message are expensive?